### ISSUE TITLE
Proxy recording

### ISSUE THREAD

**Issue by BR-** [BR-/aos_replay#8]
For self-recording, we could save a player slot (and a round trip to the server) by running as a mitm proxy.

**Comment by rakiru** [comment]
Can you spectate a full server? I had a basic pyspades replay plugin, but I didn't think you could have a spectator plus 32 other "players", which made it kinda useless.

Edit: I guess that would be a voxlap restriction (if not imaginary) anyway, so not such a big deal.

**Comment by BR-** [comment]
Good point, we'd have to strip out one of the players on playback (for voxlap client to watch, anyway, other clients might work).
 
Edit: I wonder what happens if the voxlap client is joined to a team and sent position/input data for its own ID?

**Comment by rakiru** [comment]
Well, sending a position data packet is how teleports work, iirc, so I imagine it'd "work", at least partially, but would be incredibly uncomfortable to watch.
